drive/modules/listener/http_service.bal

```diff
@@ -101,5 +104,12 @@ service class HttpService {
             check caller->respond(http:STATUS_OK);
         }
     }
+    resource isolated function get [string name](http:Caller caller, http:Request request) returns @tainted error? {
+        log:printDebug("Domain verification ...");
+        http:Response r = new();
+        r.setHeader("Content-Type", "text/html; charset=UTF-8");
+        r.setTextPayload(self.dnsVerificationFileContent);
```

Comment:
Do we need to sanitize/filter the content user sets as configurable? 
e.g: The user can provide some HTML content as the configurable.
if the front end is preventing Cross-Site Scripting (XSS). This won't be an issue.

Reply:
Added some filtration. Btw the front end sanitizes user inputs properly. Checked